straxen/misc.py

```diff
@@ -31,27 +31,28 @@
 
 
 @export
-def dataframe_to_wiki(df, float_digits=5, title='Awesome table', 
-                      force_int=tuple()):
+def dataframe_to_wiki(
+    df, float_digits=5, title='Awesome table', force_int=()):
```

Comment:
General question, should we do explicit typing whenever we change one of those functions? I guess you know what I mean, just for reference: https://docs.python.org/3/library/typing.html

Reply:
Updated.